fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -312,8 +316,25 @@ class TabsTrayFragment : AppCompatDialogFragment() {
         } else {
             tabsTrayBinding.tabWrapper
         }
+
+        val newTabFab = if (requireContext().settings().enableTabsTrayToCompose) {
+            fabButtonComposeBinding.root
+        } else {
+            fabButtonBinding.newTabButton
+        }
+
+        val behavior = BottomSheetBehavior.from(rootView)
```

Comment:
nit: The behavior could alternatively be instantiated like:
```suggestion
        val behavior = BottomSheetBehavior.from(rootView).apply {
            addBottomSheetCallback(traySheetBehaviorCallback)
            skipCollapsed = true
        }
```

Reply:
nice suggestion 👍